webrtc.html

```diff
@@ -895,6 +899,20 @@
                     <var>connection</var>.</p>
                   </li>
                   <li>
+                    <p>If <var>description</var> is set as a local description,
+                    and it creates a new SCTP association, as defined in
+                    [[!SCTP-SDP]], set the value of <var>connection</var>'s
+                    [[<code><a>sctpTransport</a></code>]] internal slot
+                    to a newly created <code><a>RTCSctpTransport</a></code>.</p>
```

Comment:
If the SctpTransport is created when calling setLocalDescription then maxMessageSize needs to be nullable since it will be null prior to setting a remote description.  Perhaps a state attribute would clarify whether the SctpTransport was new, connecting, connected or closed.